bin/install.sh

```diff
@@ -101,8 +101,26 @@ if [[ "$host" == "Win32" ]]; then
   fi
 fi
 
+function advice {
+  if [[ "$(uname -s)" == "Darwin" ]]; then
+    echo "brew install $1"
+  elif [[ "$(uname -r)" == *"ARCH"* ]]; then
+    echo "sudo pacman -S $1"
+  elif [[ "$(uname -s)" == *"Linux"* ]]; then
+    echo "sudo apt-get install $1"
+  fi
+}
```

Comment:
we can add another else branch with something like `echo "to install $1 with your package manager"`